codes/Action.ttl

```diff
@@ -69,12 +82,22 @@ idsc:DISTRIBUTE
 idsc:GRANT_USE
     a ids:Action;
     rdfs:label "grant use"@en;
-    rdfs:comment "To grant use of a resource to another party. Does *not* imply any other usage rights."@en ;
-    skos:broader <http://www.w3.org/ns/odrl/2/grantUse> ;
+    rdfs:comment "An action to be used in the count usage policy where the idsc:COUNT left operand is used."@en ;
```

Comment:
ODRL defines the Grant use action as follows: "To grant the use of the Asset to third parties."
This action is similar to Distribute action and comes along with Next Policy.
It is not clear to me why the definition here restricts this action to the Count Usage Policy.

Reply:
That is a copy-paste error, many thanks for finding them out. Already corrected. 
Did you also review all the remaining parts?